### PR TITLE
feat: bootstrap api project

### PR DESCRIPTION
**Pull request by TroyKomodo**
## Proposed changes

Bootstrap API repo

## Types of changes

What types of changes does your code introduce to 7TV?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/SevenTV/SevenTV/blob/main/CONTRIBUTING.md) doc
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

API-3

### CODE REVIEW

**Comment by TroyKomodo** [comment]
Task linked: [API-3 Bootstrap Application](https://app.clickup.com/t/4793979/API-3)

File: apps/api/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM debian:bookworm-slim
+
+WORKDIR /app
+
+# Copy the binary from the host machine
+# This assumes that you have already built the binary using the
+# `cargo build --profile release-lto` command.
+RUN --mount=type=bind,src=target/release-lto/api,dst=/mount/api \
```
[the remark this answers is not in the document]
Comment (TroyKomodo) [reply apps/api/Dockerfile:8]: So i can do the copy and the chmod in a single command

[the remark this answers is not in the document]
Comment (lennartkloock) [reply apps/api/Dockerfile:8]: Is that important?

[the remark this answers is not in the document]
Comment (TroyKomodo) [reply apps/api/Dockerfile:8]: It creates an extra layer on the docker image

[the remark this answers is not in the document]
Comment (TroyKomodo) [reply apps/api/Dockerfile:8]: I dont think its too important but its something

**Pull request merged**